sentinel.go

```diff
@@ -17,6 +18,10 @@ import (
 // FailoverOptions are used to configure a failover client and should
 // be passed to NewFailoverClient.
 type FailoverOptions struct {
+	// The network type, either tcp or unix.
+	// Default is tcp.
+	Network string
```

Comment:
Same - unix sockets don't make much sense if you want to use automatic failover.